components/home/homeSection1.tsx

```diff
@@ -12,7 +12,7 @@ const HomeSection1: FunctionComponent = () => {
         subtitle="Create your Starknet Profile by claiming your stark domain and start using it everywhere in the ecosystem."
         mobileSubtitle="Step into the future with StarkNet ID. Begin by claiming your free StarkNet identity, your passport to the blockchain world. Register your unique .stark domain to transform your presence—say goodbye to the complex addresses. Elevate your StarkNet experience today."
         buttonName="Choose your domain"
-        buttonLink="https://app.starknet.id/"
+        buttonLink="https://sepolia.app.starknet.id/"
```

Comment:
You need to make it dynamic based on wich network the user is on

Reply:
okay will do it!

Reply:
Done! As it's just a simple landing page there is no wallet connection to get the chain id, I updated the `buttonLink` according the origin url.